### ISSUE TITLE
import statement potentially not working

### ISSUE THREAD

**Issue by tobiasjacobs** [smart-data-models/data-models#66]
The import statement in the given code walkthrough

"import pysmartdatamodels as sdm"

is not working after installing with pip. What works instead is 

"from pysmartdatamodels import pysmartdatamodels as sdm"

Thanks for providing this lib; very helpful!

**Comment by albertoabellagarcia** [comment]
you are absolutely right. We are waiting for the next version to correct this in the documentation
